routing/routing_settings.cpp

```diff
@@ -84,7 +84,7 @@ RoutingSettings GetRoutingSettings(VehicleType vehicleType)
             50.0 /* m_matchingThresholdM */,
             true /* m_showTurnAfterNext */,
             measurement_utils::KmphToMps(3.0) /* m_minSpeedForRouteRebuildMpS */,
-            20.0 /* m_finishToleranceM */,
+            30.0 /* m_finishToleranceM */,
```

Comment:
Probably even 50.

Reply:
@vng should kOnEndToleranceM be modified instead?

```
  double const finishToleranceM = segmentIdx == m_routeSegments.size() - 1
                                      ? m_routingSettings.m_finishToleranceM
                                      : kOnEndToleranceM;
```

Reply:
Let's also rename these variables to make them more clear. What can you suggest?

Reply:
Google Maps finishes my route when I drive 100 meters from the point.

Reply:
```
  double const finishToleranceM = segmentIdx == m_routeSegments.size() - 1
                                      ? m_routingSettings.m_finishToleranceM
                                      : kOnEndToleranceM;
```

it looks like that m_finishToleranceM is for the final point, kOnEndToleranceM for intermediate. Why do we need two constants here?